backend/app/alembic/versions/d4867f3a4c0a_first_revision.py

```diff
@@ -23,38 +24,22 @@ def upgrade():
         sa.Column("id", sa.Integer(), nullable=False),
         sa.Column("full_name", sa.String(), nullable=True),
         sa.Column("email", sa.String(), nullable=True),
-        sa.Column("hashed_password", sa.String(), nullable=True),
+        sa.Column("profile_picture", sa.String(), nullable=True),
+        sa.Column("hashed_password", sa.String(), nullable=True, default=None),
         sa.Column("is_active", sa.Boolean(), nullable=True),
-        sa.Column("is_superuser", sa.Boolean(), nullable=True),
+        sa.Column(
+            "type", postgresql.ENUM("superuser", "student", "professor", "admin", name="user_type"), nullable=True
```

Comment:
I thought finally this wasn't going to be nullable?

Reply:
yeah, I was copy pasting alembic's autogenerated code from the revision that was giving an issue when I ran `upgrade`. Alembic seemed to set nullable.....